src/platform/forms-system/test/js/containers/FormPage.unit.spec.jsx

```diff
@@ -152,6 +156,33 @@ describe('Schemaform <FormPage>', () => {
 
     expect(router.push.calledWith('first-page'));
   });
+  it("should go back to the beginning if current page isn't found", () => {
```

Comment:
Could you please update the description for this test? It's currently the same as the one above.

Reply:
Oops! LOL thanks for catching that.